### PR TITLE
Hunter 3.4.1

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

File: modules/java/android_test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,71 @@
+if(NOT ANT_EXECUTABLE OR NOT ANDROID_EXECUTABLE OR NOT ANDROID_TOOLS_Pkg_Revision GREATER 13)
```
Comment (ruslo) [inline modules/java/android_test/CMakeLists.txt]: It seems that this file is not present in original code anymore.

File: modules/ts/misc/run_utils.py [hunk 2]
```diff
@@ -76,14 +76,26 @@ def getPlatformVersion():
     {'name': "opencv_build",             'default': None,       'pattern': re.compile(r"^OpenCV_BINARY_DIR:\w+=(.+)$")},
     {'name': "tests_dir",                'default': None,       'pattern': re.compile(r"^EXECUTABLE_OUTPUT_PATH:\w+=(.+)$")},
     {'name': "build_type",               'default': "Release",  'pattern': re.compile(r"^CMAKE_BUILD_TYPE:\w+=(.*)$")},
-    {'name': "android_abi",              'default': None,       'pattern': re.compile(r"^ANDROID_ABI:\w+=(.*)$")},
-    {'name': "android_executable",       'default': None,       'pattern': re.compile(r"^ANDROID_EXECUTABLE:\w+=(.*android.*)$")},
-    {'name': "ant_executable",           'default': None,       'pattern': re.compile(r"^ANT_EXECUTABLE:\w+=(.*ant.*)$")},
+    {'name': "git_executable",           'default': None,       'pattern': re.compile(r"^GIT_EXECUTABLE:FILEPATH=(.*)$")},
```
Comment (ruslo) [inline modules/ts/misc/run_utils.py]: I think rework needed here, there is no `git_executable` part in OpenCV 3.4.1 version for example.

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by xsacha** [comment]
Oh, I will check those again.

**Comment by xsacha** [comment]
Didn't handle that merge conflict properly so I rebased again.
That should work fine now :)

**Review by ruslo — APPROVED** [review]
Looks good, thank you.

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/opencv/releases/tag/v3.4.1-p0

**Pull request merged**